lib/components/tab.tsx

```diff
@@ -88,6 +69,12 @@ export default class Tab extends React.PureComponent<TabProps, {hovered: boolean
             color: #ccc;
           }
 
+          .tab_tab:hover .tab_icon {
+            opacity: 1;
+            transform: none;
+            pointer-events: all;
+          }
+
```

Comment:
I think it'd be better to move this block below (just above `.tab_shape`) to it's previous location, as all the tab_icon css is there, and would better reflect the fact that the main change is from `.tab_iconHovered` to `.tab_tab:hover .tab_icon` and not the properties

Reply:
I did it this way so all the `.tab` hover states are grouped together, but what you're suggesting also seems reasonable to me. I'll push the changes in a couple of minutes.